server.js

```diff
@@ -36,15 +38,39 @@ new WebpackDevServer(webpack(config), {
   stats: {
     colors: true,
   },
-}).listen(port, 'localhost', function(err) {
-  if (err) {
-    console.log(err);
-  }
+};
 
-  console.log(`Listening at localhost:${port}`);
-  if (port === 4242) {
-    console.log(
-      'You can change this default port with the environment variable PERFHTML_PORT.'
+// Allow a local file to override various options.
+if (fs.existsSync(path.join(__dirname, './webpack.local-config.js'))) {
+  try {
+    require('./webpack.local-config.js')(config, serverConfig);
+  } catch (error) {
+    console.error(
+      'Unable to load and apply settings from webpack.local-config.js'
     );
+    console.error(error);
+  }
+}
+
+new WebpackDevServer(webpack(config), serverConfig).listen(
+  port,
+  'localhost',
+  function(err) {
+    if (err) {
+      console.log(err);
+    }
+
+    console.log(`perf.html is available at: http://localhost:${port}\n`);
+    if (port === 4242) {
+      console.log(
+        'You can change this default port with the environment variable PERFHTML_PORT.\n'
+      );
+    }
+    console.log(stripIndent`
+      You can customize the webpack dev server by creating a webpack.local-config.js
+      file that exports a single function that mutates the config values:
+        (webpackConfig, serverConfig) => void
+      ------------------------------------------------------------------------------
+    `);
```

Comment:
Please change this log depending on whether `webpack.local-config.js` was used.
Case 1: it was absent so it wasn't used: output this log.
Case 2: it was present and was used: output something like `We used your local file "webpack.local-config.js" to mutate the config values`. This is useful if we have this file but forgot about it, and we have a weird error (like changing webpack.config.js and nothing changes :) ).

In addition you might also add a log something at the start of the `if` block line 45 but I believe having a differentiated here is good enough.